Sources/Basic/Process.swift

```diff
@@ -321,13 +341,23 @@ public final class Process: ObjectIdentifierProtocol {
             throw SystemError.posix_spawn(rv, arguments)
         }
 
-        if redirectOutput {
+        if outputRedirection.redirectsOutput {
+            let stdoutClosure: OutputClosure?
+            let stderrClosure: OutputClosure?
+            if case let OutputRedirection.stream(stdout, stderr) = outputRedirection {
```

Comment:
You can add properties in `OutputRedirection` to access the stdout and stderr closures. I think that would look a little cleaner